### PR TITLE
added filtration

### PR DESCRIPTION
**Pull request by t-chatoyan**
Solves https://github.com/kestra-io/docs/issues/1694

### CODE REVIEW

**Comment by wrussell1999** [comment]
Can you make the titles on the cards smaller @t-chatoyan ?
Screenshot 2024-09-06 at 14 38 48

**Comment by wrussell1999** [comment]
The search doesn't seem to work either @t-chatoyan

File: content/docs/15.how-to-guides/index.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ title: How-to Guides
 icon: /docs/icons/tutorial.svg
 ---
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/index.md:4]: We need a way to hide all the sub pages in the side menu. It should only show "How-to Guides" at all times with the only way to change guide by going back to the how-to guide home page and using the filters/search

Comment (wrussell1999) [reply content/docs/15.how-to-guides/index.md:4]: `hideSidebar: true` doesn't quite solve this as it hides How-to Guides from the menu all together. Whereas when used for the getting started folder, it kept the "Getting Started" title visible but hide all the sub pages unless selected. We want similar to that but also hide sub pages when on them

**Comment by anna-geller** [comment]
+1 to what @wrussell1999 mentioned and additionally the filters don't work well yet
- [ ] missing option to remove the filter for stage (or select "All")
- [ ] topic filter must be multi-select with the option to remove any option anytime, the same as blueprint tags
- [ ] too large title as Will said

image

**Review by wrussell1999 — APPROVED** [review]

**Pull request merged**